Source/Core/Logging/CorrelationIDGenerator.swift

```diff
@@ -1,13 +1,11 @@
 import Foundation
 
 protocol CorrelationIDManaging {
-
     func generateCorrelationID() -> String
     func destroyCorrelationID()
 }
 
 final class CorrelationIDManager: CorrelationIDManaging {
-
```

Comment:
Another interesting rule. Personally I find it useful to add a space at the start of an object and at the end:
```swift
struct MyObject {

    var isWorking = true

    func stopWorking() {
        isWorking = false
    }

}
```

but I've noticed in a few places that this now becomes:
```swift
struct MyObject {
    var isWorking = true

    func stopWorking() {
        isWorking = false
    }
}
```

Personally would want to enable dev to express themselves whether they consider that start / end spaces are useful when creating the object, instead of removing all by default, or encourage that empty line to be added by default, not removed. What do you think?